### DISCUSSION TITLE
Swagger yaml [Jun 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6026] [Discord Threads]
> 🙋‍♂️ **precognist**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### swagger is generated as a json and rendered at the moment and not available to download. Please see in n/w traffic of browser console as a workaround  
#  
> 💁 **o1lab**  
#### and what do u mean by moving the api to server  
#  
> 🙋‍♂️ **precognist**  
#### Thanks for reply. I'm trying to expand the API in the developer section of my website. (devops engineer left the team). Right now, the API only gets user data. Noco successfully generated the rest of the API, but I'm having trouble figuring out how to move it to the server. Sorry about the noob question, I'm a gamedev and just learning database backend.  
#  
> 🙋‍♂️ **precognist**  
#### ahh. i see the json file  
#  
> 🙋‍♂️ **precognist**  
#### awesome! thank you <a:HYPE:1063757967004008489>  
#  
> 💁 **o1lab**  
#### what game are u developing ? 😄    
####    
#### by moving it to server - you will have to run nocodb docker on that server  
#  
> 🙋‍♂️ **precognist**  
#### Game is called MegaCityOnline. Pc/Vr Multiplayer RPG.  
#  
> 🙋‍♂️ **precognist**  
#### Website is a test of a custom subsystem. Right now, it has all the features of a social media platform including chat, voice and video calls, live streaming, payments and subscription, groups, pages, etc.  
#  
> 🙋‍♂️ **precognist**  
#### I just have no access to any of it from the game application. And lead devops engineer left the team for a good job in Texas (USA).  
#  
> 💁 **o1lab**  
#### and what database is it  
#  
> 🙋‍♂️ **precognist**  
#### mysql  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1113520029061873774)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
